plugins/Invoke-IcingaCheckUsedPartitionSpace.psm1

```diff
@@ -51,11 +51,10 @@ function Invoke-IcingaCheckUsedPartitionSpace()
       [int]$Verbosity     = 0
    );
 
-   $DiskFree    = Get-IcingaDiskPartitions;
-   $DiskPackage = New-IcingaCheckPackage -Name 'Used Partition Space' -OperatorAnd -Verbos $Verbosity;
+   $DiskFree        = Get-IcingaDiskPartitions;
+   $DiskPackage     = New-IcingaCheckPackage -Name 'Used Partition Space' -OperatorAnd -Verbos $Verbosity;
```

Comment:
missing an `e` at -Verbose?

Reply:
Yeah in general it is missing, but is auto-completed by PowerShell.
Still fixing it would be nice :D